feature/orchestrator/src/main/java/com/simprints/feature/orchestrator/OrchestratorActivity.kt

```diff
@@ -63,6 +65,13 @@ internal class OrchestratorActivity : BaseActivity() {
                 // Some co-sync functionality depends on the exact package name of the caller app,
                 // e.g. to switch content providers of debug and release variants of the caller app
                 extras.putString(ClientApiConstants.CALLER_PACKAGE_NAME, callingPackage)
+                PreferenceManager.getDefaultSharedPreferences(this)
+                    .edit {
+                        putString(
+                            getString(com.simprints.infra.resources.R.string.preference_last_calling_package_name_key),
```

Comment:
We are usually importing the resources with `import com.simprints.infra.resources.R as IDR`, should be consistent with other classes.